src/bin/ci-generate.rs

```diff
@@ -60,7 +60,7 @@ static DEFAULT_CONF: &str = r#"
     name = ""
 
     [meson]
-    kind = "c"
+    kind = "C"
```

Comment:
@lu-zero 

We have a problem here, because we should use `c` as value to be coherent with the command line parameter, though it seems `figment` is looking for `ProjectKind` as type. Do you know how to fix this part?

Reply:
[serde rename_all](https://serde.rs/container-attrs.html#rename_all) should do: `#[serde(rename_all = "lowercase")]`

Reply:
Fixed, thanks a lot!